client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/ClassAnalyzer.java

```diff
@@ -11,22 +11,84 @@
+    private static final String JAKARTA_NEGATIVE_OR_ZERO_ANNOTATION_NAME = "jakarta.validation.constraints.NegativeOrZero";
+    private static final String JAVAX_POSITIVE_ANNOTATION_NAME = "javax.validation.constraints.Positive";
+    private static final String JAKARTA_POSITIVE_ANNOTATION_NAME = "jakarta.validation.constraints.Positive";
+    private static final String JAVAX_POSITIVE_OR_ZERO_ANNOTATION_NAME = "javax.validation.constraints.PositiveOrZero";
+    private static final String JAKARTA_POSITIVE_OR_ZERO_ANNOTATION_NAME = "jakarta.validation.constraints.PositiveOrZero";
+    private static final String JAVAX_PAST_ANNOTATION_NAME = "javax.validation.constraints.Past";
+    private static final String JAKARTA_PAST_ANNOTATION_NAME = "jakarta.validation.constraints.Past";
+    private static final String JAVAX_PAST_OR_PRESENT_ANNOTATION_NAME = "javax.validation.constraints.PastOrPresent";
+    private static final String JAKARTA_PAST_OR_PRESENT_ANNOTATION_NAME = "jakarta.validation.constraints.PastOrPresent";
+    private static final String JAVAX_FUTURE_ANNOTATION_NAME = "javax.validation.constraints.Future";
+    private static final String JAKARTA_FUTURE_ANNOTATION_NAME = "jakarta.validation.constraints.Future";
+    private static final String JAVAX_FUTURE_OR_PRESENT_ANNOTATION_NAME = "javax.validation.constraints.FutureOrPresent";
+    private static final String JAKARTA_FUTURE_OR_PRESENT_ANNOTATION_NAME = "jakarta.validation.constraints.FutureOrPresent";
+    private static final String JAVAX_NULL_ANNOTATION_NAME = "javax.validation.constraints.Null";
+    private static final String JAKARTA_NULL_ANNOTATION_NAME = "jakarta.validation.constraints.Null";
+    private static final String JAVAX_SIZE_ANNOTATION_NAME = "javax.validation.constraints.Size";
+    private static final String JAKARTA_SIZE_ANNOTATION_NAME = "jakarta.validation.constraints.Size";
+    private static final String JAVAX_DIGITS_ANNOTATION_NAME = "javax.validation.constraints.Digits";
+    private static final String JAKARTA_DIGITS_ANNOTATION_NAME = "jakarta.validation.constraints.Digits";
+    private static final String JAVAX_ENTITY_ANNOTATION_NAME = "javax.persistence.Entity";
+    private static final String JAKARTA_ENTITY_ANNOTATION_NAME = "jakarta.persistence.Entity";
+    private static final String JAVAX_TABLE_ANNOTATION_NAME = "javax.persistence.Table";
+    private static final String JAKARTA_TABLE_ANNOTATION_NAME = "jakarta.persistence.Table";
+
+    private static final String JAVAX_NOT_NULL_ANNOTATION_NAME = "javax.validation.constraints.NotNull";
+    private static final String JAKARTA_NOT_NULL_ANNOTATION_NAME = "jakarta.validation.constraints.NotNull";
+    public static final String JAVAX_COLUMN_ANNOTATION_NAME = "javax.persistence.Column";
+    public static final String JAKARTA_COLUMN_ANNOTATION_NAME = "jakarta.persistence.Column";
```

Comment:
why are those `public` while the others are `private`?

Reply:
no reason, simply a mistake.